hack/test

```diff
@@ -44,7 +44,7 @@ case $buildmode in
   buildctl build $progressFlag --frontend=dockerfile.v0 --local context=. --local dockerfile=. \
     --frontend-opt target=integration-tests \
     --frontend-opt filename=./hack/dockerfiles/test.buildkit.Dockerfile \
```

Comment:
`--frontend-opt` -> `--opt`

Reply:
@AkihiroSuda , sure, I will update all those in another PR, is that okay for you?